File: src/Entity/Contes.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
use Symfony\Component\Serializer\Annotation\Groups;
/**
 * Contes
 *
 * @ORM\Table(name="contes")
 * @ORM\Entity
 * @ORM\Entity(repositoryClass="App\Repository\ContesRepository")
 */
class Contes
{
    /**
     * @var int
     *
     * @ORM\Column(name="idConte", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     * @Groups("c:read")
     */
    private $idconte;

    /**
     * @var string
     *
     * @ORM\Column(name="titre", type="string", length=50, nullable=false)
     * @Assert\Length(min=3)
     * @Assert\NotBlank
     * @Groups("c:read")
     */
    private $titre;

    /**
     * @var string
     *
     * @ORM\Column(name="auteur", type="string", length=50, nullable=false)
     * @Assert\Length(min=3)
     * @Assert\NotBlank
     * @Groups("c:read")
     */
    private $auteur;

    /**
     * @var string
     *
     * @ORM\Column(name="contes", type="text", length=65535, nullable=false)
     * @Assert\NotBlank
     * @Assert\File(
     *     maxSize = "1024k",
     *     mimeTypes = {"application/pdf", "application/x-pdf"},
     *     mimeTypesMessage = "Please upload a valid PDF"
     * )
     */
    private $contes;

    /**
     * @var int
     * @ORM\Column(name="rate", type="integer", nullable=true)
     */
    private $rate;

    public function getIdconte(): ?int
    {
        return $this->idconte;
    }

    public function getTitre(): ?string
    {
        return $this->titre;
    }

    public function setTitre(string $titre): self
    {
        $this->titre = $titre;

        return $this;
    }

    public function getAuteur(): ?string
    {
        return $this->auteur;
    }

    public function setAuteur(string $auteur): self
    {
        $this->auteur = $auteur;

        return $this;
    }

    public function getContes(): ?string
    {
        return $this->contes;
    }

    public function setContes(string $contes): self
    {
        $this->contes = $contes;

        return $this;
    }
    /**
     * @return int
     */
    public function getRate()
    {
        return $this->rate;
    }

    /**
     * @param int $rate
     */
    public function setRate(int $rate): void
    {
        $this->rate = $rate;
    }


}

File: Mobile/src/entities/Contes.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

/**
 *
 * @author maiez
 */
public class Contes {
    private int idconte;
    private String titre;
    private String auteur;
    private String contes;
    private String rate;

    public Contes() {
    }

    public Contes(int idconte, String titre, String auteur, String contes, String rate) {
        this.idconte = idconte;
        this.titre = titre;
        this.auteur = auteur;
        this.contes = contes;
        this.rate = rate;
    }

    public Contes(String titre, String auteur, String contes, String rate) {
        this.titre = titre;
        this.auteur = auteur;
        this.contes = contes;
        this.rate = rate;
    }

    public Contes(String titre, String auteur) {
        this.titre = titre;
        this.auteur = auteur;
    }

    public int getIdconte() {
        return idconte;
    }

    public void setIdconte(int idconte) {
        this.idconte = idconte;
    }

    public String getTitre() {
        return titre;
    }

    public void setTitre(String titre) {
        this.titre = titre;
    }

    public String getAuteur() {
        return auteur;
    }

    public void setAuteur(String auteur) {
        this.auteur = auteur;
    }

    public String getContes() {
        return contes;
    }

    public void setContes(String contes) {
        this.contes = contes;
    }

    public String getRate() {
        return rate;
    }

    public void setRate(String rate) {
        this.rate = rate;
    }
    
    
}
